### PR TITLE
OEP15: Wallet Standard

### LINKED ISSUE #49: OEP: Wallet Standard
There is currently no OEP defining a wallet standard for Ontology. Different SDKs list a wallet specification:
- [Java SDK](https://github.com/ontio/ontology-java-sdk/blob/master/docs/en/Wallet_File_Specification_en.md)
- [TS SDK](https://github.com/ontio/ontology-ts-sdk/blob/master/docs/en/Wallet_File_Specification.md)
- [Dev Docs](https://dev-docs.ont.io/#/docs-en/SDKs/01-wallet-file-specification)
- [Swift SDK](https://github.com/Ryucoin/neovm-utils/blob/master/docs/ont-wallet.md)

I think that the community would benefit from having one unified wallet standard, similar to [NEP6](https://github.com/neo-project/proposals/blob/master/nep-6.mediawiki).

### PR DESCRIPTION
**Pull request by WyattMufson**
This OEP introduces a standard to unify the implementation of the wallet file specification across the various SDKs and applications. 

This will resolve #49

### CODE REVIEW

File: OEPS/OEP-9.mediawiki [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+
+A wallet file in JSON format has the following basic structure:
+
+<pre>
+{
+  "name": "MyWallet",
+  "version": "1.0",
+  "scrypt": {},
+  "defaultOntid": "did:ont:TJNxJe2Ty8eSCjCDxDPwiS78NRAn8XPTFL",
+  "defaultAccountAddress": "TJNxJe2Ty8eSCjCDxDPwiS78NRAn8XPTFL",
```
Comment (Honglei-Cong) [inline OEPS/OEP-9.mediawiki]: Although, ONT-ID can be same with account address.  
Suggest making them different in OEP, to avoid some misleading :)

Comment (WyattMufson) [reply OEPS/OEP-9.mediawiki]: Agreed - made the switch

**Comment by WyattMufson** [comment]
@kunxian-xia @AlverLyu @mooorex @Honglei-Cong this should be ready to go

**Comment by WyattMufson** [comment]
@javajoker this one too - any other comments?

**Comment by AlverLyu** [comment]
Thanks for your proposal.
We are working to draft a new standard version since current implementation is not that proper.

**Comment by mooorex** [comment]
Yes, it is a de facto implementation, but some items will make people confuse in this specification, such as the Control object includes an unexpected item "address". Besides, we have own encrypted key format rather than NEP-2 format.
it is not very flexible, such as it supports only one type of KDF.  Is it necessary to support more cryptographic algorithms, especially different KDFs in this standard?
So, in a nutshell, is it more suitable to define a new version?

**Comment by WyattMufson** [comment]
@AlverLyu @mooorex gotcha, makes sense. Would be happy to help contribute to the new standards.

**Comment by Honglei-Cong** [comment]
As discussed, close this PR and starting a new one when new wallet standard draft available?

**Comment by WyattMufson** [comment]
@Honglei-Cong We should put the conversation on the new standard here. The conversation should be open to the public - so this can act as a starting point
